static/app/views/dashboardsV2/widget/metricWidget/fields.tsx

```diff
@@ -0,0 +1,101 @@
+import {SelectValue} from 'sentry/types';
+import {defined} from 'sentry/utils';
+import {FieldValue, FieldValueKind} from 'sentry/views/eventsV2/table/types';
+
+export type MetricsColumnType = 'set' | 'counter';
+
+export enum FieldKey {
+  SESSION = 'session',
+  USER = 'user',
+}
+
+export const METRICS_FIELDS: Readonly<Record<FieldKey, MetricsColumnType>> = {
```

Comment:
We might need to fetch this from API: 
https://www.notion.so/sentry/Metrics-API-373168c15ed44ebbbca300e9a3861949#480c6303852b4c5a9fa9dfe3815363cc
Hardcoding might not be enough, especially with custom metrics.

Not a blocker for release health though 👍